src/RecoveryServices/RecoveryServices/help/New-AzRecoveryServicesAsrReplicationProtectedItem.md

```diff
@@ -685,6 +685,22 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -UseManagedDisk
+Specify whether the Azure virtual machine that is created on failover should use managed disks
+
```

Comment:
It is better to describe the accepted values here.

Reply:
> It is better to describe the accepted values here.

Done